tests/e2e/test_minimal_io_and_set_baseline.py

```diff
@@ -0,0 +1,60 @@
+"""
+OCS-685
+
+Demonstrate that the ceph configuration can handle a minimal level of I/O
+and establish a baseline for comparision of later I/O tests.
+"""
+import pytest
+import logging
+import json
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from tests.fixtures import (
+    create_rbd_storageclass, create_rbd_pod, create_pvc, create_ceph_block_pool,
+    create_rbd_secret, create_project
+)
+
+logger = logging.getLogger(__name__)
+
+
+def do_io_actions(io_type, io_info):
+    """
+    Display fio statistics and stash fio results somewhere.
+
+    Args:
+        io_type (str):  either "Read" or "Write"
+        io_info (dict): results of fio run.
```

Comment:
io_info (dict) ---> this is the not the complete results of fio run. In the test case, from the test case, we are passing subset of fio results ( like read , write ). Please change the description accordingly.

Reply:
Okay.